xgi/algorithms/connected.py

```diff
@@ -114,6 +122,39 @@ def number_connected_components(H):
     return num_cc
 
 
+def largest_connected_component(H):
+    """
+    A function to find the largest connected component of a hypergraph.
+
+    Parameters
+    ----------
+    H: Hypergraph object
+        The hypergraph of interest
+
+    Returns
+    -------
+    set
+        the largest connected component of the hypergraph.
```

Comment:
Start with capital T. Also, explicitly say that this is a set of nodes? Contrary to largest_connected_hypergraph